bedrock/externalpages/templates/externalpages/pocket/firefox/new-tab-learn-more.html

```diff
@@ -0,0 +1,97 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block pocket_css %}
+ {{ css_bundle('pocket-new-tab') }}
+{% endblock %}
+
+{% block content %}
+<header class="new-tab-header">
+    <div class="mzp-c-logo mzp-t-logo-xs mzp-t-product-family">Firefox</div>
```

Comment:
Nit: 2 space indentation for HTML files (tip: installing the [editorconfig](https://editorconfig.org/) plugin for your code editor will help with auto formatting things like this for you).